app/controllers/api/v1/soap_api_controller.rb

```diff
@@ -5,7 +5,9 @@ class Api::V1::SoapApiController < Api::V1::BaseController
     return user.valid_password?(password)
   }
 
-  before_action :load_adapter
+  before_action do
```

Comment:
I stumbled upon the same thing, I think it can also be overcome a bit more easily with: `before_action :load_adapter, except: :_generate_wsdl`